powerquery-docs/Connectors/Funnel.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Power Query Funnel connector
-description: Provides basic information, prerequisites, and information on how to connect to your Funnel data, along with a list of known issues and limitations.
-author: bezhan-msft
+description: Provides basic information, prerequisites, and information on how to connect to your data, along with a list of known issues and limitations.
+author: Funnel
```

Comment:
This should remain bezhan-msft _or_ a GitHub username from Funnel (not Funnel itself)